Plugins/Locales/itIT.lua

```diff
@@ -302,76 +302,76 @@ L.fadeTimeDesc = "Dopo quanti secondi il messaggio deve scomparire, in secondi"
 -- Nameplates.lua
 --
 
---L.nameplates = "Nameplates"
---L.testNameplateIconBtn = "Show Test Icon"
---L.testNameplateIconBtn_desc = "Creates a test icon for you to test your current display settings with on your targeted nameplate."
```

Comment:
You removed the full stop from nearly every single "desc" (description) translation here.